<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchRooms</class>
 <widget class="QDialog" name="SwitchRooms">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>381</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>381</width>
    <height>166665</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>381</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mouvement interne au service</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/lit</normaloff>:/lit</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="5,5,0,1">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>363</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Chambre d'origine</string>
     </property>
     <widget class="QComboBox" name="comboChDep">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelChDep">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboSectDep">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Secteur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Chambre</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>363</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Chambre de destination</string>
     </property>
     <widget class="QComboBox" name="comboChArr">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelChArr">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboSectArr">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>58</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Chambre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>46</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Secteur</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ChambrePrise" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;b&gt;La chambre de destination n'est pas libre.&lt;/b&gt;&lt;br/&gt; Que voulez vous faire ?&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="options">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,2,2">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boutonValid">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Valider</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boutonAnul">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
